docker-compose.yml

```diff
@@ -5,8 +5,7 @@ services:
     build:
       context: "./discord-bot"
       dockerfile: ".docker/Dockerfile"
-    env_file: "discord-bot/.env"
-
+    env_file: "discord-bot/.env.docker"
```

Comment:
why ?

Reply:
vi` minh de? app trong container nen app -> db di qua docker network -> db_url cua? no dung` docker alias
thuc. ra e docker exec vao` trong container de? chay. migrate, bla bla cung~ dc nhung de? ra them de? tien. 
ben ngoai` e dung` .env ben trong dung` .env.docker

Reply:
ok makes sense to me